packages/react/ds/src/tabs/tab-item.tsx

```diff
@@ -49,9 +48,20 @@ export function TabItem({
           gi-underline
           gi-text-slate-300"
       >
-        <Link href={href} noVisited noColor noUnderline>
+        <a
```

Comment:
I think this should be a `<button>` rather than a `<a>` . I don't see a use case where the menu item will be a link

Reply:
Use case whenever you need to use url as state.

Reply:
As @Thurfjell wrote, we can use the anchor to change the page state. 
If you don't use the `href` the `<a>` is acting as a `<span>` but with focus.

Reply:
Makes sense